packages/client/hmi-client/src/components/workflow/ops/interventions/tera-interventions-drilldown.vue

```diff
@@ -0,0 +1,395 @@
+<template>
+	<tera-drilldown
+		:node="node"
+		@on-close-clicked="emit('close')"
+		@update-state="(state: any) => emit('update-state', state)"
+		@update:selection="onSelection"
+	>
+		<template #sidebar>
+			<tera-slider-panel
+				v-model:is-open="isSidebarOpen"
+				content-width="360px"
+				header="Intervention policies"
+			>
+				<template #content>
+					<div class="m-3">
+						<div class="flex flex-column gap-1">
+							<tera-input v-model="filterInterventionsText" placeholder="Filter" />
+						</div>
+						<ul v-if="!isFetchingPolicies">
```

Comment:
Can we not just check if the `interventionsPolicyList.value` is populated instead?
This would us allow to completely remove this `isFetchingPolicies` and this helper function: `fetchInterventionPolicies` which cleans things up slightly

Reply:
I _think_ this would show an infinite loader if there are no policies related to the model yet